src/cascadia/TerminalConnection/AzureConnectionStrings.h

```diff
@@ -26,5 +26,6 @@ const auto tokensRemoved = L"Tokens removed!\r\n";
 const auto exitStr = L"Exit.\r\n";
 const auto authString = L"Authenticated.\r\n";
 const auto internetOrServerIssue = L"Could not connect to Azure. You may not have internet or the server might be down.\r\n";
+const auto oldCredentialsFlushedMessage = L"Authentication parameters changed. You'll need to log in again.\r\n";

```

Comment:
There was a todo to localize all of these, right?

Reply:
There certainly is. In my head it's blocked behind #2146 for ScopedResourceLoader